Winium/Winium.Mobile.Common/ConnectionInformation.cs

```diff
@@ -1,10 +1,10 @@
-﻿namespace Winium.StoreApps.Common
+﻿namespace Winium.Mobile.Common
 {
     public class ConnectionInformation
     {
         #region Public Properties
 
-        public const string FileName = ".Winium.StoreApps.ConnectionInformation.json";
+        public const string FileName = ".Winium.Mobile.ConnectionInformation.json";
```

Comment:
I think we should look into backwards compatibility and support both names for a couple of releases. If new file is not found, then it should fallback to the old one and log warning message.